endoreg_db/models/metadata/sensitive_meta_logic.py

```
@@ -1158,24 +1041,19 @@ def _map_gender_string_to_standard(gender_str: str) -> Optional[str]:
             return standard
     return None
 
-
-def _create_anonymized_record(
-    instance: "SensitiveMeta",
-    DEFAULT_ANONYMIZED=None,
-    DEFAULT_ANONYMIZED_DATE=timezone.make_aware(datetime(1900, 1, 1)),
-) -> None:
+def _create_anonymized_record(instance: "SensitiveMeta", DEFAULT_ANONYMIZED=None, DEFAULT_ANONYMIZED_DATE=timezone.make_aware(datetime(1900, 1, 1))) -> None:
```

Ruff (0.14.6)

Do not perform function call `timezone.make_aware` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)